lib/cursor.js

```diff
@@ -41,8 +41,11 @@ class Cursor {
       if (document.pointerLockElement !== renderer.domElement) return
       this.buttons[e.button] = true
     })
+    bot.lastPlaced = 4 // ticks since last placed
```

Comment:
if this isn't needed in bot, you could have this variable in this. instead

Reply:
how can i make a variable across init and update event?

Reply:
constructor*

Reply:
it's a class so you just use `this.lastPlaced`